<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_part3_2_modif</class>
 <widget class="QWidget" name="fct_part3_2_modif">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Modification</string>
   </property>
  </widget>
  <widget class="QComboBox" name="argentModif">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="modifierButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Médaille de bronze</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numéro épreuve</string>
   </property>
  </widget>
  <widget class="QComboBox" name="numEpModif">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Médaille d'argent</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Médaille d'or</string>
   </property>
  </widget>
  <widget class="QComboBox" name="orModif">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="bronzeModif">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Modification d'une épreuve</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fct_part3_2_modif">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>571</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>modifierButton</sender>
   <signal>clicked()</signal>
   <receiver>fct_part3_2_modif</receiver>
   <slot>modifierRes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>modifierRes()</slot>
 </slots>
</ui>
